### PR TITLE
BaSP-TG-33: fixed the data model of tasks, forced to change test

### PR DESCRIPTION
**Pull request by agusting19**

### CODE REVIEW

File: src/tests/employees.test.js [hunk 1]
```diff
@@ -188,7 +188,7 @@ describe('create an employee', () => {
     expect(response.status).toBe(201);
     expect(response.body.message).toBe('Employee created');
     expect(response.body.error).toBeFalsy();
```
Comment (ArianaMaldonado) [inline src/tests/employees.test.js:190]: It is a bad practice to have more than one expect by test. You need to check all these situations and separate them in different tests.

File: src/tests/tasks.test.js [hunk 2]
```diff
@@ -111,28 +113,34 @@ describe('POST /api/tasks', () => {
...
+      taskName: 'Test Task',
+      startDate: '2022-05-17T16:55:32.654+00:00',
+      workedHours: 33,
+      description: 'description',
+      status: 'Done',
     });
     expect(response.status).toBe(200);
     expect(response.body.error).toBe(false);
   });
   test('Update a task, bad id', async () => {
```
Comment (ArianaMaldonado) [inline src/tests/tasks.test.js:125]: Wrong id*

File: src/tests/tasks.test.js [hunk 3]
```diff
@@ -111,28 +113,34 @@ describe('POST /api/tasks', () => {
...
+      startDate: '2022-05-17T16:55:32.654+00:00',
+      workedHours: 33,
+      description: 'description',
+      status: 'Done',
     });
     expect(response.status).toBe(404);
     expect(response.body.message).toBe('Task not found');
     expect(response.body.error).toBe(true);
   });
   test('Update a task, bad id format', async () => {
```
Comment (ArianaMaldonado) [inline src/tests/tasks.test.js:137]: Wrong id format*

File: src/validation/tasks.js [hunk 4]
```diff
@@ -2,35 +2,59 @@ import Joi from 'joi';
...
+
   return next();
 };
 
 const validateUpdate = (req, res, next) => {
   const schema = Joi.object({
-    taskDate: Joi.date(),
+    taskName: Joi.string().min(3),
+    startDate: Joi.date(),
     workedHours: Joi.number(),
```
Comment (ArianaMaldonado) [inline src/validation/tasks.js:36]: What about decimal numbers? And negative numbers?

File: src/models/Tasks.js [hunk 5]
```diff
@@ -4,12 +4,12 @@ const { Schema } = mongoose;
...
   {
-    taskId: { type: Schema.Types.ObjectId },
-    taskDate: { type: Date, required: true },
+    taskName: { type: String, required: true },
+    startDate: { type: Date, required: true },
     workedHours: { type: Number, required: true },
     description: { type: String, required: true },
+    status: { type: String, required: true },
   },
-  { timestamps: true },
```
Comment (Javier-Cervera) [inline src/models/Tasks.js:12]: I think `timestamps` could be useful. `startDate` of a task can be different from when it's created.

**Review by NicolasFontana — APPROVED** [review]
Everything looks good, approved PR

**Review by Javier-Cervera — APPROVED** [review]
Thanks for the employee validation fix!

**Pull request merged**